jpirumvaa/jpirumva
# #X11I7CfO/20 Creating Blog Page

#### What does this PR do?
The blog page is a part of the portfolio where visitors can view the posts posted by the owner of the blog. It is made responsive to enable people with medium and small-size screens to access the best it well.
#### Description of Task to be completed?
Have the following parts of the page:
Header
Blog summary
One blog details
Add comment section
Recent comments
Footer
All of these parts should be responsive.
#### How should this be manually tested?
To test this part, download the codes, and open them into the browser. Check if every part has complete information. Make sure you test it on small and medium-size screens to check its responsiveness.
#### Any background context you want to provide?
The content on the page are the placeholder. Don't complain much about them.
#### What are the relevant pivotal tracker stories?
[https://trello.com/c/kV5yqVMf/27-blog-page]()
#### Screenshots (if appropriate)
![blog](https://user-images.githubusercontent.com/50476344/88195414-4ec92600-cc40-11ea-947b-14e746243f47.png)
![blogresp](https://user-images.githubusercontent.com/50476344/88195475-630d2300-cc40-11ea-9a5d-662a3906e52f.png)
